draft-ietf-quic-transport.md

```diff
@@ -3170,11 +3170,21 @@ delayed retransmissions from the peer. For Initial and Handshake packets,
 a max_ack_delay of 0 is used. The sender uses the receiver's `max_ack_delay`
 value in determining timeouts for timer-based retransmission, as detailed in
 Section 5.2.1 of {{QUIC-RECOVERY}}.
+The max_ack_delay needs to be set so that at least several samples can
+be generated per RTT to estimate the path RTT.
 
-An ACK frame SHOULD be generated for at least every second ack-eliciting packet.
-This recommendation is in keeping with standard practice for TCP {{?RFC5681}}.
+A QUIC receiver can generate one ACK frame for every received
+ack-eliciting packet.  TCP recommends
+that a receiver generates an ACK corresponding to every second MSS of
+received data, Section 4.2 of {{?RFC5681}}, however {{?RFC3449}} also
+notes the need for, and deployment of, methods to further
+reduce the number of TCP ACKs in networks with asymmetric paths.
 
-In order to assist loss detection at the sender, an endpoint SHOULD send an ACK
+An ACK frame SHOULD be generated for at least every tenth ack-eliciting
```

Comment:
There's no mention here of ACKing more frequently for N initial packets to avoid reducing the increase in slow-start.

Reply:
Sure you could add that rule also. When do you think this helps... for RTT << ACK_delay?

Reply:
We saw no appreciable benefit for RTT>>ACK_Delay, however this really does seem good practice, and our draft (draft-fairhurst-quic-ack-scaling) proposes this change:
      The receiver can provide a higher rate of acknowledge for the
      first 100 ACK-eliciting packets, where it acknowledges at least
      every second received ACK-eliciting packet.  A suitable method
      might send an ACK frame for every two received ack-eliciting
      packets for the first 100 received packets if max_ack_delay time
      has passed since the oldest unacknowledged data was received.
Would that resolve that point?